src/DelegateDecompiler.Tests/NestedExpressionsTests.cs

```diff
@@ -178,5 +194,107 @@ public void TestFuncWithStaticMethodClosure()
                 ints => ints.SingleOrDefault(i => i == M2())
             );
         }
+
+        [Test, Ignore("Difference is expected")]
+        public void TestQueryableBoundAsVariable()
+        {
+            IQueryable<int> query = Enumerable.Empty<int>().AsQueryable().Where(i => i >= 0);
+            Test<Func<IQueryable<int>, IQueryable<int>>>(
+                ints => Enumerable.Empty<int>().AsQueryable().Where(i => i >= 0),
+                ints => query
+            );
+        }
+
+        [Test, Ignore("Difference is expected")]
+        public void TestQueryableRefFromField()
+        {
+            Test<Func<IQueryable<int>, IQueryable<int>>>(
+                ints => Enumerable.Empty<int>().AsQueryable().Where(i => i >= 0),
```

Comment:
I think this test is incorrect. It should be `ints => fQref1.Where(i => i >= 0)`